src/CMakeLists.txt

```diff
@@ -109,9 +109,10 @@ if(BUILD_CLIENT AND ENABLE_SOUND)
 	endif()
 endif()
 
-option(ENABLE_TOUCH "Enable Touchscreen support" FALSE)
+option(ENABLE_TOUCH "Enable touchscreen by default" FALSE)
```

Comment:
IMO we should just get rid of this option
or is there an use case for it?

Reply:
the Android build sets it
mobile Linux distributions might want to set it, but it will be obsolete if we somehow implement proper auto-detection

Reply:
android is a solved issue: we have an ifdef in defaultsettings.cpp

Reply:
needed until we get auto-detection. I have this in followup MR.